src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs

```diff
@@ -405,16 +405,16 @@ public async Task CanPackageForAllSupportedContainerRIDs(string dockerPlatform,
 
         // Build the image
         Registry registry = new(ContainerHelpers.TryExpandRegistryToUri(DockerRegistryManager.BaseImageSource));
-
+        var isWin = rid.StartsWith("win");
         ImageBuilder? imageBuilder = await registry.GetImageManifestAsync(
             DockerRegistryManager.RuntimeBaseImage,
-            DockerRegistryManager.Net8PreviewImageTag,
+            isWin ? DockerRegistryManager.Net8PreviewWindowsSpecificImageTag : DockerRegistryManager.Net8PreviewImageTag,
```

Comment:
This is required because the dotnet containers have changed to no longer include windows images in the manifest list.